infrastructure/constructs/lambda_config.py

```diff
@@ -11,3 +16,9 @@
     RETENTION_DAYS = aws_logs.RetentionDays.ONE_YEAR
 else:
     RETENTION_DAYS = aws_logs.RetentionDays.THREE_MONTHS
+
+PACKAGING_DIR = tempfile.mkdtemp(dir=BACKEND_DIRECTORY, prefix=".lambda_out_")
+
+
+def delete_packaging_dir(packaging_dir: str = PACKAGING_DIR) -> None:
+    shutil.rmtree(packaging_dir)
```

Comment:
Since this is a single line we could possibly inline it as `lambda: shutil.rmtree(PACKAGING_DIR)`.